src/utils/localization/fetchLocale.ts

```diff
@@ -0,0 +1,34 @@
+import { locales } from "../locale";
+
+/**
+ * Get supported locale code from raw user input
+ * Exported for testing purposes.
+ *
+ * @param lang
+ * @private
+ * @returns {string}
+ */
+export function getSupportedLocale(lang = ""): string {
```

Comment:
Do we need a default here? I'm not sure in which scenario we wouldn't have a locale defined.

Reply:
nope, we dont.